packages/block-editor/README.md

```diff
@@ -854,6 +854,10 @@ _Parameters_
 -   _props_ `Object`: Optional. Props to pass to the element. Must contain the ref if one is defined.
 -   _options_ `Object`: Optional. Inner blocks options.
 
+### useReusableBlocksRenameHint
+
+Undocumented declaration.
```

Comment:
I guess `useReusableBlocksRenameHint()` needs a JS doc block.

Does this need to be part of the public API as well or could it be private?

I'm not sure myself 🤔

Reply:
Same as `ReusableBlocksRenameHint` it shouldn't be part of the public API - what is the best way to indicate that it has to be exported in order to be accessible in the reusable blocks pacakge?

Reply:
Should it be part of the private APIs? https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/private-apis.js#L27

?

Then imported via `import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';`

The  reusable blocks package would need an unlock file, e.g., https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/lock-unlock.js

Reply:
It is a bit of extra work for something that isn't really of any use to anybody, but let's do that just to be safe

Reply:
> wp.blockEditor.useReusableBlocksRenameHint()

Agree that it's pretty useless to the wider community, but that's also a good argument to hide it behind the private API: to avoid maintain backwards compat for a function that doesn't have much utility in the broader context.

It's still accessible via 
 `wp.blockEditor.useReusableBlocksRenameHint()`, thanks to the fact that the blockEditor package exports everything that is exported from /components via `export * from './components'; `

I always bet on the assumption that someone out there will use it 😄

Reply:
just double checked and not available via `wp.blockEditor.useReusableBlocksRenameHint` now that it is moved to the private APIs.